app/serializers/api/product_serializer.rb

```diff
@@ -49,10 +49,17 @@ class Api::CachedProductSerializer < ActiveModel::Serializer
   has_many :images, serializer: Api::ImageSerializer
   has_one :supplier, serializer: Api::IdSerializer
 
+  #return an unformatted descripton
   def description
     strip_tags object.description
   end
 
+  #return a sanitized html description
+  def description_html
+    d = sanitize(object.description, options = {tags: "p, b, strong, em, i"})
```

Comment:
@oeoeaio The pre-save sanitization remains unchanged, so it's using whatever we're using for all the other textarea fields throughout the admin area. I havn't properly looked into how that's currently done, but I assume there's some basic sanitizing, escaping, etc.

My thinking with this was to leave in the original **description** attribute unchanged, which just uses `strip_tags`. This can still be used to render plain-text descriptions without the HTML tags or styling, which could be useful in other places (like an RSS feed, or a preview snippet of the first 10 words of the description). 

Since the output of the new **description_html** attribute will be rendered directly as html, it needs to be marked with `html_safe` before it gets to the view, so I thought I'd make sure it was definitely safe before doing so.

From my limited knowledge of web security: theoretically if an SQL injection vulnerability was found someone could inject malicious data into the database and pre-save sanitation wouldn't necessarily help. My understanding of the way Rails handles data means that further problems would be fairly limited, unless it happened to be in a field that gets marked with `html_safe` before hitting the browser, in which case very bad things could happen with XSS...

That was my thinking for the use of `html_safe` preceded by very explicit sanitization.